features/step_definitions.rb

```diff
@@ -246,6 +246,30 @@
 
 #
 
+Then(%r!^I should see date "(.*)" in "(.*)" unless Windows$!) do |text, file|
+  step %(the "#{file}" file should exist)
+  regexp = Regexp.new(text)
+  if Jekyll::Utils::Platforms.really_windows? && !dst_active?
+    expect(file_contents(file)).not_to match regexp
```

Comment:
Because the regex `2013-04-10T03:14:00-04:00` **will match** `actual value` when:
  - The Platform is Windows and **in an active DST Zone**.
  - The Platform is not Windows.

So this step is:
```ruby
if !Jekyll::Utils::Platforms.really_windows?
  expect(file_contents(file)).to match regexp
elsif Jekyll::Utils::Platforms.really_windows? && dst_active?
  expect(file_contents(file)).to match regexp
else
  expect(file_contents(file)).not_to match regexp
end
```
which could be shortened to ( but may hamper readability ):
```ruby
if (!Jekyll::Utils::Platforms.really_windows?) ||
  (Jekyll::Utils::Platforms.really_windows? && dst_active?)
  expect(file_contents(file)).to match regexp
else
  expect(file_contents(file)).not_to match regexp
end
```
which could be alternatively written in the negative as:
```ruby
if Jekyll::Utils::Platforms.really_windows?) && !dst_active?
  expect(file_contents(file)).not_to match regexp
else
  expect(file_contents(file)).to match regexp
end
```